plonky2x/src/frontend/vars/byte.rs

```diff
@@ -237,6 +256,43 @@ impl<L: PlonkParameters<D>, const D: usize> Zero<L, D> for ByteVariable {
     }
 }
 
+impl<L: PlonkParameters<D>, const D: usize> Add<L, D> for ByteVariable {
+    type Output = Self;
+
+    fn add(self, rhs: Self, builder: &mut CircuitBuilder<L, D>) -> Self::Output {
+        let mut bits = array![_ => builder.init::<BoolVariable>(); 8];
```

Comment:
I believe here, you want to use `builder.init_unsafe` (which is something John just added), since you're going to be just setting elements of `bits` from safe variables. @jtguibas can you check me on this?